website/src/content/support-policy.mdx

```diff
@@ -27,21 +27,22 @@ Tooling and packages related to Custom Applications are released and published t
 
 Releases are documented in both a `CHANGELOG.md` file and in the GitHub release pages in the following repositories:
 
-- Changelog
-  - [Merchant Center Application Kit](https://github.com/commercetools/merchant-center-application-kit/blob/master/CHANGELOG.md)
-  - [UIKit](https://github.com/commercetools/ui-kit/blob/master/CHANGELOG.md)
-- Release notes
-  - [Merchant Center Application Kit](https://github.com/commercetools/merchant-center-application-kit/releases)
-  - [UIKit](https://github.com/commercetools/ui-kit/releases)
+- [Merchant Center Application Kit](https://github.com/commercetools/merchant-center-application-kit)
+  - [`CHANGELOG.md`](https://github.com/commercetools/merchant-center-application-kit/blob/master/CHANGELOG.md)
+  - [Release notes](https://github.com/commercetools/merchant-center-application-kit/releases)
+
+- [UIKit](https://github.com/commercetools/ui-kit)
+  - [`CHANGELOG.md`](https://github.com/commercetools/ui-kit/blob/master/CHANGELOG.md)
+  - [Release notes](https://github.com/commercetools/ui-kit/releases)
 
 All published packages to NPM are listed in the following scoped packages:
 
-- [`@commercetools-frontend`](https://www.npmjs.com/org/commercetools-frontend)
-- [`@commercetools-uikit`](https://www.npmjs.com/org/commercetools-uikit)
+- Merchant Center Application Kit [`@commercetools-frontend`](https://www.npmjs.com/org/commercetools-frontend)
+- UIKit [`@commercetools-uikit`](https://www.npmjs.com/org/commercetools-uikit)
```

Comment:
Hmm the npm scopes are not necessarily related to the github repository.

Reply:
good point. will address this